content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+<br />
+
+```bash
+terraform init
+```
+
+The `init` command above will download plugins and providers from the **provider.tf** file. Next, preview the resources Terraform will create. 
+<br />
+
+```bash
+terraform plan
+```
+
+The output displays the resources Terraform will create in an actual implementation. Lastly, finish creating all resources using the following command. 
+<br />
+
+```bash
+terraform apply -auto-approve
+```
+
+It will take up to 20 mins to provision a cluster. See a sample successful deplyment in the visual below. 
+While the deployment is in progress, you can switch to the Palette to validate the in-progress deployment. 
+
+<br /> <br />
+
+
+## Check the In-Progress Deployment
+Log back into the [Palette console](https://console.spectrocloud.com/), and navigate to the **Profile** section in the left **Main Menu**. If the terraform deployment is successful, you will see the newly created Profile, as shown in the visual below. 
+
+<br /> <br />
+
+![Visual of the successful Profile in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_verify-profile.png)
+
+<br /> <br />
+
+## Validate
+
+Navigate to the **Clusters** section in the Palette console's left **Main Menu** and view the cluster deployment in progress. After the Palette finishes provisioning the cluster, the status will turn to **Running** and **Healthy**. You can access the application on the exposed port 8080 for the **Hello Universe** service. 
+
+<br /> <br />
+
+![Visual of the cluster health.](/tutorials/create-pack/registries-and-packs_create-pack_cluster-health.png)
+
+<br /> <br />
+
+The application will look similar to the visual below. 
+
+<br /> <br />
+
+![Visual of the successful accessing the application using the load balancer URL.](/tutorials/create-pack/registries-and-packs_create-pack_success.png)
+
+<br /> <br />
+
+Congratulations, you've successfully deployed the *Hello Universe* application to the cluster. 
+
+<br /> <br />
+
+## Cleanup
+
+To delete the resources, switch back to the "tutorials" container, and run the following command from within the **terraform/pack-tf** directory:
+<br />
+
+```bash
+terraform destroy -auto-approve
+```
+<br /> 
+
+Wait for the resources to clean up and ensure the successful resource deletion in the Palette console as well. 
+
+</Tabs.TabPane>
+
+</Tabs>
+
+<br /> <br /> 
+
+# Wrap-Up
+
+In this tutorial, you created a custom add-on pack to package the *Hello Universe* application and deployed it to a cluster. First, you learned to create the pack using the manifest files. Then, along the learning journey, you learned to set up a private registry server and publish the pack to that registry. In the latter half of the tutorial, you configured the registry server in the Palette console so that you can access the pack to create a Profile. Later, you created a Profile and deployed it to a cluster using the Palette console or Terraform. 
```

Comment:
```suggestion
In this tutorial, you learned how to create a custom pack using manifest files. You packaged up an application in a custom pack that you pushed to a private registry server that you later added to Palette.

Next, you created a cluster profile that included all the core infrastructure layers, such as the OS, Kubernetes distribution, and more. You also added your custom pack to the cluster profile so your application could be deployed to a Kubernetes cluster. 

Packs are the building blocks of cluster profiles. It's what allows you to customize all your Kubernetes clusters. You can use different packs to create different cluster profiles for specific purposes. This is the value Palette provides. You can ensure all Kubernetes deployments contain all the required dependencies and applications without having to develop complicated deployment scripts. All you need to do is maintain the cluster profiles. 

We encourage you to check out the following resources to learn more about Palette.
```

A conclusion should include a high-level summary that is easy to follow and highlights our product's business value. This is an example of a conclusion that emphasizes the value of packs, which is the main element of this tutorial.  I didn't mention Terraform because that's not important. It's more important that the user understands the value of the pack and why you would want to create a custom pack. Lastly, add some links below that you feel the reader should check out to continue their learning journey.